### ISSUE TITLE
Address Gerrit comments

### ISSUE THREAD

**Issue by jakar** [jakar/qt-heif-image-plugin#15]
@zzag made some very good comments [here](https://codereview.qt-project.org/c/qt/qtimageformats/+/236077) awhile ago. They should be addressed.

**Comment by zzag** [comment]
Yeah, one thing that I'm really concerned about is that image sequences and image collections are two different things.
